docs/_posts/2016-01-11-admin-manual.md

```diff
@@ -115,14 +115,18 @@ and populate it with the following information:
       username="tgaclims"
       password="tgaclims"/>
       <Parameter name="miso.propertiesFile" value="file:${catalina.home}/conf/Catalina/localhost/miso.properties" override="false"/>
-      <Parameter name="miso.name" value="Test"/>
+      <Parameter name="miso.instanceName" value="Your MISO instance name"/>
+      <!-- uncomment if using Runscanner -->
+      <!--
+      <Parameter name="runscanner.configFile" value="/etc/runscanner.json" override="false"/>
+      -->
     </Context>
 
 Make sure the database path in `ROOT.xml` is correct for your install:
 
     url="jdbc:mysql://your.database.server:3306/lims"
 
-Also, set the `miso.name` parameter to something to help distinguish testing
+Also, set the `miso.instanceName` parameter to something to help distinguish testing
```

Comment:
`miso.instanceName` comes from `miso.properties` now. Should be removed from `ROOT.xml`